src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleDescribable.java

```diff
@@ -158,6 +158,8 @@ public static String[] getValues() {
 
     public static final String ANSIBLE_ENCRYPT_EXTRA_VARS = "ansible-encrypt-extra-vars";
 
+    String ANSIBLE_YAML_DATA_SIZE = "ansible-yaml-data-size";
```

Comment:
this should be static and final

Reply:
Interface attributes in Java are by default public , static and final. Indicate them is redundant.